src/api/routes/projects.js

```diff
@@ -1,25 +1,37 @@
 const ProjectsService = require('../route-services/projects');
 
+const {
+  expressAuthorizationMiddleware,
+  expressAuthenticationOnlyMiddleware,
+} = require('../../utils/authMiddlewares');
+
 const projectsService = new ProjectsService();
 
 module.exports = {
-  'projects#update': (req, res, next) => {
-    projectsService.updateProject(req.params.projectUuid, req.body)
-      .then((data) => res.json(data))
-      .catch(next);
-  },
-  'projects#delete': (req, res, next) => {
-    projectsService.deleteProject(req.params.projectUuid)
-      .then((data) => res.json(data))
-      .catch(next);
-  },
-
-  'projects#get': (req, res, next) => {
-    projectsService.getProjects().then((response) => res.json(response)).catch(next);
-  },
-  'projects#getExperiments': (req, res, next) => {
-    projectsService.getExperiments(req.params.projectUuid)
-      .then((response) => res.json(response)).catch(next);
-  },
-
+  'projects#update': [
+    expressAuthenticationOnlyMiddleware,
+    (req, res, next) => {
+      projectsService.updateProject(req.params.projectUuid, req.body)
+        .then((data) => res.json(data))
+        .catch(next);
+    }],
+  'projects#delete': [
+    expressAuthorizationMiddleware,
+    (req, res, next) => {
+      projectsService.deleteProject(req.params.projectUuid)
+        .then((data) => res.json(data))
+        .catch(next);
+    }],
+  'projects#get': [
+    expressAuthenticationOnlyMiddleware,
```

Comment:
Wrong middleware?

Reply:
This is also auth only because this endpoint gets all projects, not a specific `experimentId` so we are not checking for any authorization yet. We might do this in the future, but for now I don't think we're doing it.